modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/ObjectEntriesFDSPropsTransformer.tsx

```diff
@@ -3,24 +3,75 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-interface ObjecEntriesDataRendererProps {
+import {openModal} from 'frontend-js-web';
+
+import ModalDeleteObjectEntry from '../ObjectEntry/ModalDeleteObjectEntry';
+
+interface ItemData {
+	actions: ItemDataDelete;
+	id: number;
+}
+interface ItemDataDeleteProps {
+	href: string;
+	method: string;
+}
+interface ObjectEntriesDataRendererProps {
 	value: {
 		key: string;
 		name: string;
 		name_i18n: string;
 	}[];
 }
 
-export default function ObjectEntriesFDSPropsTransformer({
-	...otherProps
-}) {
+type ActionDataProps = {
```

Comment:
@Artartur I don't think it's necessary to create this type and the others below. Just declare them at the moment of usage.

Reply:
Done!